source/game/kart/KartJump.cc

```diff
@@ -23,6 +23,32 @@ KartJump::KartJump(KartMove *move) : m_move(move) {
 /// @addr{0x80575AA8}
 KartJump::~KartJump() = default;
 
+/// @addr{0x805764FC}
+void KartJump::calcRot() {
+    m_angleDelta *= m_angleDeltaFactor;
+    m_angleDelta = std::max(m_angleDelta, m_properties.angleDeltaMin);
+    m_angleDeltaFactor -= m_angleDeltaFactorDec;
+    m_angleDeltaFactor = std::max(m_angleDeltaFactor, m_properties.angleDeltaFactorMin);
+    m_angle += m_angleDelta;
+    m_angle = std::min(m_angle, m_finalAngle);
+
+    switch (m_type) {
+    case TrickType::KartFlipTrickZ: {
+        EGG::Vector3f angles = EGG::Vector3f(0.0f, 0.0f, -(m_angle * DEG2RAD) * m_rotSign);
+        m_rot.setRPY(angles);
+    } break;
+    case TrickType::FlipTrickYLeft:
+    case TrickType::FlipTrickYRight: {
+        EGG::Vector3f angles = EGG::Vector3f(0.0f, m_angle * DEG2RAD * m_rotSign, 0.0f);
+        m_rot.setRPY(angles);
```

Comment:
```suggestion
        m_rot.setRPY(EGG::Vector3f(0.0f, 0.0f, -(m_angle * DEG2RAD) * m_rotSign));
        break;
    case TrickType::FlipTrickYLeft:
    case TrickType::FlipTrickYRight:
        m_rot.setRPY(EGG::Vector3f(0.0f, m_angle * DEG2RAD * m_rotSign, 0.0f));
```
I'm 80% sure this allows the removal of the brackets for the switch statement.